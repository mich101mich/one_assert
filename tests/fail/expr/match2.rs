fn main() {
    // the other match.rs file checks for types and stops checking, so this file is needed to check for later errors
    one_assert::assert!(match 0 {});
}
