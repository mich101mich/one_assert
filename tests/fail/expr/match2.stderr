error[E0004]: non-exhaustive patterns: type `i32` is non-empty
 --> tests/fail/expr/match2.rs:3:31
  |
3 |     one_assert::assert!(match 0 {});
  |                               ^
  |
  = note: the matched value is of type `i32`
help: ensure that all possible cases are being handled by adding a match arm with a wildcard pattern as shown
  |
3 ~     one_assert::assert!(match 0 {
4 +         _ => todo!(),
5 ~     });
  |
